Packs/CommonTypes/IndicatorTypes/reputation-asn.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "ASN",
```

Comment:
Can you change the file name to be with big letters? (ASN and not asn)

Reply:
This is our schema, like `cve`, `cidr`, `url` etc. filenames are in lower case

Reply:
![image](https://user-images.githubusercontent.com/24267931/183827956-8d76f95c-8e99-464d-85d9-443e8372863d.png)